### ISSUE TITLE
[Accessibility] Word wrap does not work in diff view (F7 and Shift+F7)

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#13434]
CC -isidorn and -meganrogge

Although the recent VSCode update introduced accessibility improvement for word wrap, which used to not work with screen reader mode, I found that the word wrap setting does not take any effect in F7/Shift+F7 (diff view) when  screen reader mode is on.

It would be great if word wrap also works in diff view mode as well.

-- System Info

Version: 1.75.0-insider (system setup)
Commit: f076d553210e9ea65e7bccbfa2f7a786c10fcab2
Date: 2022-12-09T09:11:59.691Z
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: Yes
